src/Morpho.sol

```diff
@@ -377,11 +377,16 @@ contract Morpho is IMorpho {
         uint256 badDebtShares;
         if (position[id][borrower].collateral == 0) {
             badDebtShares = position[id][borrower].borrowShares;
-            uint256 badDebt = badDebtShares.toAssetsUp(market[id].totalBorrowAssets, market[id].totalBorrowShares);
-            market[id].totalSupplyAssets -= badDebt.toUint128();
-            market[id].totalBorrowAssets -= badDebt.toUint128();
-            market[id].totalBorrowShares -= badDebtShares.toUint128();
             position[id][borrower].borrowShares = 0;
+
+            uint128 totalBorrowAssets = market[id].totalBorrowAssets;
```

Comment:
Is this stack assignment necessary? It goes against minimalist conventions we took for Blue (of relying on the compiler for optimizations)

Reply:
Should I @MathisGD ?